package.json

```diff
@@ -22,7 +22,7 @@
     "@google-cloud/firestore": "^0.19.0",
     "@sentry/node": "^4.3.0",
     "@tensorflow/tfjs-node": "^0.3.0",
-    "@zeit/webpack-asset-relocator-loader": "0.4.1",
+    "@zeit/webpack-asset-relocator-loader": "0.4.2",
```

Comment:
Please add integration tests for esm, consolidate, fluent-ffmpeg, and if possible the TS dist bug.

Reply:
Sure, I've added the integration tests.